src/views/edit/edit-provider-view.vue

```diff
@@ -33,7 +33,7 @@
             v-if="id"
             :label="$t('component.general.uuid.label')"
             :value="uuid"
-            path="/provider"
+            path="/#/provider"
```

Comment:
This does not work for creating an URL which points to the REST API for example. We do not want a link to this client.